docs/policies.rst

```diff
@@ -54,55 +59,64 @@ In pseudocode, what the ``SimplePolicy`` above does is:
     if we were previously listening:
         return a canned response
     else:
-        we must have just said something, so let's Listen again
+        we must have just said something, so let's listen again
 
 
 Note that the policy itself is stateless, and all the state is carried by the ``tracker`` object.
 
 
 Creating Policies from Stories
-------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Writing rules like in the SimplePolicy above is not a great way to build a bot, it gets messy fast & is hard to debug.
 If you've found Rasa Core, it's likely you've already tried this approach and were looking for something better.
-A good next step is to use our story framework to build a policy by giving it some example conversations.
-We won't use machine learning yet, we will just create a policy which memorises these stories. 
 
-We can use the ``MemoizationPolicy`` to do this.
+The second important method of any policy is ``train(...)``:
 
-Here is the ``train`` method training the policies class:
+.. doctest::
 
-.. literalinclude:: ../rasa_core/agent.py
-   :pyobject: Agent.train
+    def train(self,
+          training_trackers,  # type: List[DialogueStateTracker]
+          domain,  # type: Domain
+          **kwargs  # type: **Any
+          ):
+    # type: (...) -> None
+    """Trains the policy on given training trackers."""
```

Comment:
this function is missing a code block, should be at least `pass`

Reply:
in the superclass it is `raise NotImplementedError("Policy must have the capacity to train.")`

What is better `pass` or `raise`?

Reply:
mhm lets use the raise

Reply:
then in predict_action_probabilities as well?
I will use the code directly from methods from `policy.py`